cluster/common.sh

```diff
@@ -682,6 +682,21 @@ EOF
 INITIAL_ETCD_CLUSTER: $(yaml-quote ${INITIAL_ETCD_CLUSTER})
 EOF
     fi
+    if [ -n "${APISERVER_FEATURE_CONFIG:-}" ]; then
+      cat >>$file <<EOF
+APISERVER_FEATURE_CONFIG: $(yaml-quote ${APISERVER_FEATURE_CONFIG})
```

Comment:
Is there a reason we'll want to pass different features to different components? Why not declare that all components will get the same set of features?